src/schema/v2/sale/index.ts

```diff
@@ -110,13 +110,19 @@ export const SaleType = new GraphQLObjectType<any, ResolverContext>({
               "List of artwork IDs to exclude from the response (irrespective of size)",
           },
           status: { type: SaleArtworkStatus },
+          cached: {
+            description:
+              "When this is true and there is no access token present, allow a loader that caches to be used.",
+            type: GraphQLBoolean,
```

Comment:
should be `new GraphQLNonNull(GraphQLBoolean)` (very much dislike these 3-valued bools)

Reply:
OH I thought this was the type